packages/livekit-rtc/src/rpc.ts

```diff
@@ -0,0 +1,58 @@
+// TODO: This implementation is only a prototype
+//       The final version will use a protocol types where possible
+export class RpcRequest {
+  id: string;
+  method: string;
+  payload: string;
```

Comment:
in the future I'd like to be able to extend payload to support streaming large payloads - haven't thought enough about whether this interface would be backwards compatible when that happens or not

Reply:
I am thinking a separate type would be cleaner. Something like `RpcRequestStream`.

Reply:
makes sense - do we think it would be ok for it to be a whole parallel API (e.g. `registerStreamingRpcMethod`, `performStreamingRpcMethod`)?

Reply:
That would be my take. The separation is very clear that way.